lang/ghc/Portfile

```diff
@@ -112,14 +112,6 @@ if {${name} eq ${subport}} {
                     "OPT=${prefix}/bin/opt-mp-${ghc_llvm_version}" \
                     "PATH=${workpath}/bin:${haskell_cabal.cabal_root}/bin:$env(PATH)"
 
-    # https://gitlab.haskell.org/ghc/ghc/-/issues/23477
-    post-extract {
-        copy        ${worksrcpath}/docs/index.html.in \
-                    ${worksrcpath}/docs/index.html
-        reinplace   "s|@LIBRARY_ghc_VERSION@|${version}|g" \
-                    ${worksrcpath}/docs/index.html
-    }
-
```

Comment:
Could you explain the removal of these lines? The GHC issue you referenced is still open.